lib/SearchAndSort/SearchAndSort.js

```diff
@@ -1184,6 +1187,9 @@ class SearchAndSort extends React.Component {
             <FormattedMessage id={messageKey} values={{ count: source.totalCount() }} />
           )}
           {!isSourceLoaded && (
+            customPaneSubText?
```

Comment:
Run lint before pushing up your branch to avoid these kinds of errors.

Reply:
Done.